int32 transponderID
int32 commandID
string data